compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/PlatformContextImpl.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.window
+
+import androidx.compose.ui.input.InputMode
+import androidx.compose.ui.platform.DefaultInputModeManager
+import androidx.compose.ui.platform.EmptyViewConfiguration
+import androidx.compose.ui.platform.PlatformContext
+import androidx.compose.ui.platform.TextToolbar
+import androidx.compose.ui.platform.ViewConfiguration
+import androidx.compose.ui.platform.WindowInfo
+import androidx.compose.ui.text.input.PlatformTextInputService
+import androidx.compose.ui.unit.dp
+
+internal class PlatformContextImpl(
```

Comment:
It was supposed to be a provider itself, not just something empty that receives everything by parameters.

```suggestion
internal class IOSPlatformContextImpl(
```

Reply:
Renamed.
I think it is better to leave it as is now